### ISSUE TITLE
CIMultiDict: No overload variant of "get" of "Mapping" matches argument type "str"

### ISSUE THREAD

**Issue by ewjoachim** [aio-libs/multidict#687]
## Long story short

`CIMultiDict().get("str")`  fails in mypy

## Expected behaviour

```python
import multidict

a = multidict.CIMultiDict({"a": "b"})

a.get("Hey")
```
Should pass in mypy

## Actual behaviour

```
foo.py:5: error: No overload variant of "get" of "Mapping" matches argument type "str"
foo.py:5: note: Possible overload variants:
foo.py:5: note:     def get(self, key: istr) -> Optional[str]
foo.py:5: note:     def [_T] get(self, istr, Union[str, _T]) -> Union[str, _T]
```
## Steps to reproduce

Write the above code in foo.py, run `mypy foo.py` with the most recent multidict & mypy

## Your environment

Python 3.8.10
```
mypy==0.931
multidict==6.0.1
```

**Comment by asvetlov** [comment]
Fixed by #688
Sorry for the inconvinience.

**Comment by ewjoachim** [comment]
Ow thank you for the quick fix !
